spec/services/price_policies/time_based_price_calculator_spec.rb

```diff
@@ -225,6 +225,14 @@
           it "uses usage_rate as rate" do
             is_expected.to eq(cost: 120, subsidy: 0)
           end
+
+          it "calls the correct strategy" do
+            expect_any_instance_of(
+              PricePolicies::Strategy::PerMinute
+            ).to receive(:calculate)
+
+            subject
+          end
```

Comment:
Given in line 169 we're defining `subject` using `PricePolicies::Strategy::PerMinute`, is this spec useful?

Reply:
Line 169 is just defining a let like declaration named `discount`, I had to change that due to the refactor